pages/common/pixiecore.md

```diff
@@ -0,0 +1,8 @@
+# pixiecore
+
+> Pixiecore is a tool to manage network booting of machines.
+> More information: <https://github.com/danderson/netboot/tree/master/pixiecore>.
+
+- Boot with netboot.xyz on the same network:
```

Comment:
```suggestion
- Boot with `netboot.xyz` on the same network:
```
I don't have any idea what this command does. Probably also because I don't know anything about that topic. However, is there any chance this could be explained a little bit more?

Reply:
Also, which machine does this boot? I'm assuming that it's not the one you're entering the command on.....

Reply:
Hi @marchersimon  @sbrl , sorry I didn't reply earlier.

Pixiecore is a PXE boot server: it serves a bootable image. Then any machine on the network which chooses to boot from the network (typically you would choose this option in the boot menu) would load that image and boot with it.

[Netboot.xyz](https://netboot.xyz/) is a bootable image which consists essentially in a menu that lets you choose and download a bootable image from the internet (be it an Ubuntu installer, an RHEL installer, etc… or a diagnostic image).

If you can suggest pithy ways of making this one line more precise, I'm all for it, but I find it difficult myself.

Reply:
So you execute that command on the server right? How about: "Create a PXE boot server which provides a `netboot.xyz` boot image"? Or am I wrong here?

Reply:
Maybe “Starts a PXE […]” would be better? Other that this, sounds good to me.

Reply:
(I just pushed the “Starts” version, let me know if this is fine)